kratos/utilities/time_discretization.cpp

```diff
@@ -0,0 +1,101 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "time_discretization.h"
+
+namespace Kratos {
+namespace TimeDiscretization {
+
+void BDF1::ComputeBDFCoefficients(const double DeltaTime, std::vector<double>& rCoefficients) const
+{
+
+    if (rCoefficients.size() != 2) rCoefficients.resize(2);
+
+    rCoefficients[0] =  1.0/DeltaTime;
+    rCoefficients[1] = -1.0/DeltaTime;
+}
+
+void BDF2::ComputeBDFCoefficients(const double DeltaTime, std::vector<double>& rCoefficients) const
+{
+    if (rCoefficients.size() != 3) rCoefficients.resize(3);
+
+    const double denom = 1.0/(2.0*DeltaTime);
```

Comment:
you need to implement this for the case of variable time steps...

Reply:
hmh sure, no problem
I didn't do it so far to not bloat it up right from the start

I will add it to the list of requirements

but in general, do you agree with what I am proposing here?

Reply:
done